src/routes/Swap.tsx

```diff
@@ -283,36 +282,38 @@ export default function Swap() {
                         }}
                     >
                         <Switch>
-                            <Match when={channelOpenResult()?.failure_reason}>
-                                <img
-                                    src={megaex}
-                                    alt="fail"
-                                    class="w-1/2 mx-auto max-w-[30vh] flex-shrink"
-                                />
-
-                                <p class="text-xl font-light py-2 px-4 rounded-xl bg-white/10">
+                            <Match
+                                when={channelOpenResult()?.failure_reason}
+                            >
+                                <MegaEx />
+                                <LargeHeader centered>
+                                    {channelOpenResult()?.failure_reason
+                                        ? channelOpenResult()?.failure_reason?.message
+                                        : ""}
+                                </LargeHeader>
+                                {/*Error hint needs to be added for possible failure reasons
+                                <p class="text-xl font-light py-2 px-4">
                                     {
                                         channelOpenResult()?.failure_reason
                                             ?.message
                                     }
-                                </p>
+                                </p>*/}
                             </Match>
-                            <Match when={true}>
-                                <img
-                                    src={megacheck}
-                                    alt="success"
-                                    class="w-1/2 mx-auto max-w-[30vh] flex-shrink"
-                                />
-                                <AmountCard
-                                    amountSats={
-                                        channelOpenResult()?.channel?.balance?.toString() ||
-                                        ""
-                                    }
-                                    reserve={
-                                        channelOpenResult()?.channel?.reserve?.toString() ||
-                                        ""
-                                    }
+                            <Match when={channelOpenResult()?.channel}>
+                                <MegaCheck />
+                                <div class="flex flex-col justify-center">
+                                <LargeHeader centered>
+                                    Swap Initiated
+                                </LargeHeader>
+                                <p class="text-2xl text-center">
+                                    +{channelOpenResult()?.channel?.balance.toString() ?? "0"} sats will be added to your Lightning balance
```

Comment:
need to format this string

![Screenshot 2023-07-19 at 3 18 00 PM](https://github.com/MutinyWallet/mutiny-web/assets/543668/86759822-9799-4ecb-b773-d61e81eaf925)